### ISSUE TITLE
Scaling or pre-sizing images

### ISSUE THREAD

**Issue by Cinemacloud** [ktock/container2wasm#192]
How do I resize or configure the size at build? I didn't see much mention of that but perhaps entirely a docker concern and question?

**Comment by ktock** [comment]
Is this duplication of #191 ?

**Comment by Cinemacloud** [comment]
More or less but curious how I can at least tweak storage size (through Docker?) before its implemented in c2w itself.. or not possible?

**Comment by ktock** [comment]
>  tweak storage size (through Docker?)

Could you elaborate what "storage" means? The storage usage by docker during build? Or the output Wasm image size? Or the disk size of the container(VM) running on browser?

**Comment by Cinemacloud** [comment]
yes, practically I was just updating the packages and it quickly ran out of space and stopped working.

**Comment by ktock** [comment]
We rely on `docker build` for generating wasm images so it sounds like Docker's concern. If it's ok to remove your Docker build cache, Docker provides `docker builder prune` for removing build cache and increasing free space. https://docs.docker.com/engine/reference/commandline/builder_prune/

**Comment by Cinemacloud** [comment]
So c2w has no practical limits other than Docker prescribes via images and containers?

**Comment by ktock** [comment]
Yes. If needed, we can fix c2w to optionally use some flags of `docker build` for limiting resource usage: https://docs.docker.com/engine/reference/commandline/build/#options

**Comment by Cinemacloud** [comment]
Not sure if it worked but commenting out entire /Users/admin/ainima/node_modules/.pnpm/browser-with-fingerprints@1.5.4/node_modules/browser-with-fingerprints/src/plugin/mutex/index.js 

file seems to have made it go away

**Comment by Cinemacloud** [comment]
Just curious if you were able to give hardware scaling params into build yet and also could you put together a demo using Neko browser which is docker based already.
